sktime/clustering/partitioning/_lloyds.py

```diff
@@ -264,11 +265,19 @@ def _check_params(self, X: np.ndarray) -> None:
         self._random_state = check_random_state(self.random_state)
 
         if isinstance(self.init_algorithm, str):
-            self._init_algorithm = self._init_algorithms.get(self.init_algorithm)
-        else:
+            try:
```

Comment:
I recommed not using try/except if the condition can be explicitly checked.
I would recommed `isinstance(init_algorithm, np.ndarray)`

Reply:
thanks!